docs/default.nix

```diff
@@ -209,17 +209,37 @@ let
 
                 maintainersText =
                   if maintainers == [ ] then
-                    "This module has no [dedicated maintainers](../../modules.md#maintainers)."
+                    "This module has no [dedicated maintainers](../../modules.md#maintainers).\n"
                   else
                     ''
                       This module is maintained by ${renderedMaintainers},
                       pingable via: `${ghHandles}`.
                     '';
+
+                # Render homepages as hyperlinks in readme
+                #
+                # Takes the `homepages` attribute set from `meta.nix` and
+                # combines it with homepages extracted from  the `packages`
+                # list, also from `meta.nix`.
+                homepages = metadata.${module}.homepages or { };
+
+                renderHomepage = name: url: "[${name}](${url})";
+
+                renderedHomepages = joinItems (lib.mapAttrsToList renderHomepage homepages);
```

Comment:
Ensure that modules with a single homepage are not declared as

```nix
homepages = { "<MODULE>" = "<HOMEPAGE>"; };
```

by leveraging `lib.throwIf` and making sure that attribute sets contain `> 1` elements.